src/backend/common/CommandBufferStateTracker.cpp

```diff
@@ -164,10 +164,6 @@ namespace backend {
     };
```

Comment:
A new validation aspect should be added "SUBPASS" which should gate most operations that are currently gated on "RENDERPASS"